web/src/hooks/useUserLocation.ts

```diff
@@ -35,7 +35,7 @@ export const useUserLocation = (): UserLocationType => {
         // eslint-disable-next-line -- check needed for tests
         if (coords) {
           const { latitude, longitude } = coords
-          setUserCoordinates([latitude, longitude])
+          setUserCoordinates([longitude, latitude])
```

Comment:
I think it should be the other way round (at least that is the default, right?) At least [latitude, longitude] seems to be the default, e.g. on wikipedia.

Reply:
true...
https://docs.mapbox.com/mapbox-gl-js/api/geography/
Then our poi coordinates are also wrong